apps/www/components/sidebar-nav.tsx

```diff
@@ -55,7 +55,7 @@ export function DocsSidebarNavItems({
             target={item.external ? "_blank" : ""}
             rel={item.external ? "noreferrer" : ""}
           >
-            {item.title}
+            <span className="hover:underline">{item.title}</span>
```

Comment:
This should be a `group-hover:underline` i belive as hovering over the entire row should make it underline

Reply:
PR updated for this fix @imMatheus .